.github/workflows/test2.yml

```diff
@@ -23,6 +29,8 @@ on:
       - 'metrics/**'
       - 'docs/**'
       - '**.md'
+      - '!waku/v2/**.nim'
+      - '!tests/v2/**.nim'
```

Comment:
Same here. This was deliberate. We want all tests to be executed when merging into the master branch. Please, remove this too.